backend/framework/qlf/dashboard/bokeh/qlf_plot.py

```diff
@@ -161,7 +161,7 @@ def metric_table(metricname, comments, keyname,  curexp='xxx', refexp='xxx', obj
     return style + title + header + tblines + end
 
 
-def mtable(qa, data, objtype=['XXELG', 'XXSTAR']):
+def mtable(qa, data, comments, objtype=['OTYPE ?', 'OTYPE ?']):
```

Comment:
Is this `comments` variable necessary?

You are not using it so let's remove it

Reply:
done